llvm/lib/ProfileData/SampleProfReader.cpp

```diff
@@ -791,6 +798,19 @@ bool SampleProfileReaderExtBinaryBase::useFuncOffsetList() const {
   return false;
 }
 
+std::error_code
+SampleProfileReaderExtBinaryBase::read(const DenseSet<StringRef> &FuncsToUse,
```

Comment:
Now we need to check whether a profile is already loaded, before attempt to load.